aircraft.py

```diff
@@ -179,7 +179,7 @@ def setup(self, Nclimb, Ncruise, enginestate, eng, fitDrag, BLI = False, Nmissio
                             self.LG['x_n'] <= self.fuse['l_{nose}'],
                             TCS([self.LG['x_m'] >= self.fuse['x_{wing}']]),
                             self.LG['x_m'] <= self.wing['\\Delta x_{AC_{wing}}'] + self.fuse['x_{wing}'],
-                            xhpesys == 1.1*self.fuse['l_{nose}'],
+##                            xhpesys == 1.1*self.fuse['l_{nose}'],
```

Comment:
Also @mayork I am finding that the wingbox location variables do not stay tight... what a pain.